uktrade/data-store-service
# World bank Bulk L0 load

To test:

Run the following:
 - `./manage.py dev db --create_tables`
 - `./manage.py dev datafiles_to_db_by_source --dit.eu_country_membership`
 - `./manage.py dev datafiles_to_db_by_source --comtrade.country_code_and_iso`
 - `./manage.py dev datafiles_to_db_by_source --world_bank.bulk`

Check DB to see if data loads into the L0 and L1 db tables within the `world_bank.bulk` schema

Note if you leave the command to run it will take 4 hours and at least 50GB of space so kill it after a while.
